packages/atomic/src/components/facets/atomic-date-facet/atomic-date-facet.tsx

```diff
@@ -101,7 +102,18 @@ export class AtomicDateFacet implements InitializableComponent, BaseFacetState {
     );
   }
 
+  public get totalNumberOfResults() {
+    return this.facetState.values.reduce(
+      (accum, value) => accum + value.numberOfResults,
+      0
+    );
+  }
```

Comment:
Starting a thread here since it is easier to track responses compared to the `conversation` page.

I just read JF's comment. What do the JSUI dynamic range facets currently do when:

1. There are no results
2. There are results, but a range has a count of 0
3. There are results, a range has a count of 0, and it is selected.

Reply:
1. The facet is hidden.
2. It's hidden from the facet's results.
3. I'm not really sure how to get to this state given 2, since if a range has no results it doesn't appear and you can never select down a facet to zero results.

Reply:
For #3 it show the selected range with 0 result in order to let the user deselect it IIRC

Reply:
For the third state, select a range, and then perform a query that gives a count of 0. As JF said, we want to keep the range displayed in this case to allow a user to deselect it.